lib/ast.js

```diff
@@ -1241,6 +1242,21 @@ var AST_True = DEFNODE("True", null, {
     value: true
 }, AST_Boolean);
 
+/* -----[ Yield ]----- */
+
+var AST_Yield = DEFNODE("Yield", "value is_star", {
```

Comment:
`value` should be renamed `expression` to be consistent with other AST nodes like `AST_Unary`.